src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/validation/FormAValidator.java

```diff
@@ -2,61 +2,43 @@
 
 import org.springframework.stereotype.Component;
 
-import java.util.ArrayList;
-import java.util.Collection;
+import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
 
 import static java.util.Arrays.asList;
-import static uk.gov.hmcts.reform.bsp.common.model.validation.BulkScanValidationPatterns.CCD_PHONE_NUMBER_REGEX;
 
 @Component
 public class FormAValidator extends BulkScanFormValidator {
 
     private static final List<String> MANDATORY_FIELDS = asList(
-        "PetitionerFirstName",
-        "PetitionerLastName"
+        "claimingExemptionMIAM",
+        "familyMediatorMIAM",
+        "applicantAttendedMIAM"
     );
 
     private static final Map<String, List<String>> ALLOWED_VALUES_PER_FIELD = new HashMap<>();
 
     static {
-        // obviously will be adjusted to suit Form A
-        ALLOWED_VALUES_PER_FIELD.put("D8LegalProcess", asList("Divorce", "Dissolution", "Judicial (separation)"));
+        List<String> yesNoValues = Collections.unmodifiableList(asList("Yes", "No"));
+        ALLOWED_VALUES_PER_FIELD.put("claimingExemptionMIAM", yesNoValues);
+        ALLOWED_VALUES_PER_FIELD.put("familyMediatorMIAM", yesNoValues);
+        ALLOWED_VALUES_PER_FIELD.put("applicantAttendedMIAM", yesNoValues);
     }
 
     public List<String> getMandatoryFields() {
         return MANDATORY_FIELDS;
     }
 
     @Override
-    protected Map<String, List<String>> getAllowedValuesPerField() {
-        return ALLOWED_VALUES_PER_FIELD;
+    protected List<String> runPostProcessingValidation(Map<String, String> fieldsMap) {
+        return Collections.emptyList();
     }
 
     @Override
-    protected List<String> runPostProcessingValidation(Map<String, String> fieldsMap) {
-        List<String> errorMessages = Stream.of(
-            validateFieldMatchesRegex(fieldsMap, "D8PetitionerPhoneNumber", CCD_PHONE_NUMBER_REGEX)
-        )
-            .flatMap(Collection::stream)
-            .collect(Collectors.toList());
-
-        return errorMessages;
+    protected Map<String, List<String>> getAllowedValuesPerField() {
+        return ALLOWED_VALUES_PER_FIELD;
     }
 
-    private static List<String> validateFieldMatchesRegex(Map<String, String> fieldsMap, String fieldKey, String validationRegex) {
-        List<String> validationMessages = new ArrayList<>();
-
-        if (fieldsMap.containsKey(fieldKey)) {
-            String valueToValidate = fieldsMap.get(fieldKey);
-            if (!valueToValidate.matches(validationRegex)) {
-                validationMessages.add(fieldKey + " is not in a valid format");
-            }
-        }
-        return validationMessages;
-    }
```

Comment:
Do you know if this is a part of the common library now?

Reply:
looks like not at the moment, but will need to add it once validation against regex comes in play